src/integrationTest/resources/db/migration/common/V1_201__active_event_handler_test_data.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO darts.event_handler (evh_id, event_type, event_sub_type, event_name, handler, created_ts, last_modified_ts, last_modified_by, active) VALUES (1001, 'ActiveTestType', 'ActiveTestSubType', 'Old Description', 'StandardEventHandler', current_timestamp, current_timestamp, 0, false);
```

Comment:
Is it possible to restart the `evh_seq` sequence in `darts-api/src/main/resources/db/migration/common/V1_101__insert_event_handler_reference_data.sql`
It seem to have gotten lost in the flyway scripts refactor?